test/test_prototype_transforms_functional.py

```diff
@@ -1337,3 +1345,16 @@ def _compute_expected_bbox(bbox, output_size_):
         else:
             expected_bboxes = expected_bboxes[0]
         torch.testing.assert_close(output_boxes, expected_bboxes)
+
+
+@pytest.mark.parametrize("device", cpu_and_gpu())
+@pytest.mark.parametrize("output_size", [[4, 3], [4], [7, 7]])
+def test_correctness_center_crop_segmentation_mask(device, output_size):
+    def _compute_expected_segmentation_mask(mask, output_size):
+        return F.center_crop_image_tensor(mask, output_size)
```

Comment:
I'm not really happy about this given that we don't seem to have a reference test that makes sure `F.center_crop_image_tensor`. Thus, if that had a bug, we would propagate it to this test. 

Plus, we now use exactly the same implementation for the test and kernel and thus are unable to detect any bug with this test.

Maybe we can have something like

```py
class TestCorrectnessCenterCrop:
    def _compute_expected_image_or_segmentation_mask(self, input, output_size):
        # manual implementation that does not use our tensor kernels
        # IMO, it would be fine to use `center_crop_image_pil` here.
        pass
    
    def test_image(self):
        pass
    
    def test_segmentation_mask(self):
        pass
```

Reply:
I think the problem here is with the `_compute_expected_segmentation_mask` implementation which is exactly the same as `F.center_crop_segmentation_mask`. IMO, this way of testing does not bring any value. 

As for restructuring with `TestCorrectnessCenterCrop`, @pmeier it can be maybe done in your functional tests refactor PR, not here.

Let's move forward with this. If we abandoned somehow the idea of mocking `F.center_crop_image_tensor` op and just checking the args, now let's recode `_compute_expected_segmentation_mask` such that we compute in another way a center crop and check output vs expected results.